index.js

```diff
@@ -59,7 +57,7 @@ async function updateGist(tweet) {
           filename: `@${twitterHandle} - ${timeAgo} ago | ❤ ${
             tweet.favorite_count
           } | 🔁 ${tweet.retweet_count}`,
-          content: wrap(tweet.text)
+          content: wrapAnsi(tweet.text, 48, { hard: true })
```

Comment:
What's the reason for wrapping at 48 characters?
I'm consistently getting 59 characters to be visible on a single line:
![image](https://user-images.githubusercontent.com/4658208/87803945-31035780-c821-11ea-9b77-d5e607fe37f6.png)

☝️  This is 59 "W"s past the breakpoint where the pinned gist is at its widest.

Reply:
🤦‍♂️ Now I think about it, the max length of a line depends on your font. Sorry for the trouble! Do you have a recommended value for this (or rather it's just `62`?

Reply:
I think it'd be best to leave it as-is for now. Thanks for the PR! 😄